### DISCUSSION TITLE
Keyboard Move Sprite

### DISCUSSION THREAD

**Discussion by josiahdowdy** [mattjennings/svelte-pixi#19] [Q&A] [answered]
I'm enjoying svelte-pixi a lot, I got Spine sprites working on it too which is a big plus. I saw the click and drag sprite example and I'm trying to reference it to use keyboard movement. Looking at the api I see "click, mousedown, tap, touchstart, etc" but not anything for a keyboard. What am I missing here? Thanks for the help!

**Comment by mattjennings** [comment, marked as answer]
Hey @josiahdowdy, glad you are enjoying it!

Pixi.js doesn't offer anything for handling keyboard events, but you can set up a [listener on the window](https://developer.mozilla.org/en-US/docs/Web/API/Document/keydown_event) for `keydown` or similar in your Svelte components. The events emitted from the SveltePixi components are [directly forwarded from the underlying Pixi.js instance](https://pixijs.download/release/docs/PIXI.DisplayObject.html#mousedown), which largely only includes mouse/pointer-based events.

**Reply by josiahdowdy** [reply]
Thanks @mattjennings! I'll update here when I get something working.
